examples/girder/src/App.vue

```diff
@@ -0,0 +1,55 @@
+<template lang="pug">
+v-app
+  app-toolbar(
+    :title='title'
+  )
+    template(slot='right')
+      girder-user-button(
+        @login='login',
+        @user='logout'
+      )
+
+  p.user-info.text-xs-center
+    span(v-if='$girder.user') You are logged in as #[b {{ $girder.user.login }}].
+    span(v-else) You are not logged in.
+
+  girder-user-dialog(
+    :form.sync='userForm',
+    v-model='userDialog'
+  )
+</template>
+
+<style lang="stylus">
```

Comment:
Should this be `scoped`?

Reply:
In this case no because it's setting global properties.  It's also the top-level application; however, I think reusable components themselves should only use scoped css.